packages/feature_flags/index.ts

```diff
@@ -0,0 +1,35 @@
+import * as raw_flags from './flags.json';
+import type { FlagList as FormalFlagList } from './flag_schema';
+
+export const flags: FormalFlagList = raw_flags;
+
+export function checkFlag(flagName: keyof typeof raw_flags) {
+   const flagValue = flags[flagName];
+
+   if (flagValue === true) {
+      return true;
+   }
+
+   const enableFlag = 'enable-' + flagName;
+
+   if (typeof flagValue === 'object') {
+      if (flagValue.userToggle) {
+         return localStorage.getItem(enableFlag) === 'true';
```

Comment:
Do we need to check if localstorage is available? Also, should this be `window.localstorage`?

```suggestion
         return window.localStorage.getItem(enableFlag) === 'true';
```